src/components/ChatHeader/ChatHeader.jsx

```diff
@@ -1,59 +1,59 @@
-import { useState } from 'react';
-import PropTypes from 'prop-types';
 import { useSelector } from 'react-redux';
-import { getUser } from '@/redux-store/selectors.js';
-import CountryInfo from '../CountryInfo/CountryInfo';
+import { useState } from 'react';
+import { getUser } from '@/redux-store/selectors';
 import { ChatHeaderStyled } from './ChatHeaderStyled';
+import CountryInfo from '../CountryInfo/CountryInfo';
```

Comment:
We chould rename sucj paths as this '../CountryInfo/CountryInfo' to this way '@/components//CountryInfo/CountryInfo';
It doesn`t mistake.

Reply:
done